core/src/wallet/ripple/explorers/RippleLikeBlockchainExplorer.h

```diff
@@ -62,25 +62,14 @@ namespace ledger {
             std::string sender;
             Option<Block> block;
             uint64_t confirmations;
+            Option<uint64_t> destinationTag;
+
             std::vector<api::RippleLikeMemo> memos;
 
             RippleLikeBlockchainExplorerTransaction() {
                 confirmations = 0;
             }
 
-            RippleLikeBlockchainExplorerTransaction(const RippleLikeBlockchainExplorerTransaction &cpy) {
```

Comment:
Why is that removed?

Reply:
This copy constructor was not doing anything, according to Pierre. He asked me to remove it.